hooks/useAuth/useAuth.tsx

```diff
@@ -59,27 +37,56 @@ interface AuthProviderProps {
 }
 
 export const AuthProvider = ({ children }: AuthProviderProps) => {
-  const [user, setUser] = useState<User | null>(null);
+  const [user, setUser] = useState<User>(defaultUser);
   const [loading, setLoading] = useState<boolean>(true);
+  const [currentCohortYear, setCurrentCohortYear] =
+    useState<number>(defaultCohortYear);
```

Comment:
```suggestion
    useState<Cohort["academicYear"]>(defaultCohortYear);
```
can make reference to the cohort academic year type (which is still number)